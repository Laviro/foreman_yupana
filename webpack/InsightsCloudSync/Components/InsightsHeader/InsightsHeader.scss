.insights-header {
  margin-bottom: '15px';
  overflow: auto;
}
