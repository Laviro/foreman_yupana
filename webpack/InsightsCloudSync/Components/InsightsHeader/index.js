import React from 'react';
import { Text, TextVariants } from '@patternfly/react-core';
import { translate as __ } from 'foremanReact/common/I18n';
import './InsightsHeader.scss';

const InsightsHeader = () => (
  <div className="insights-header">
    <Text component={TextVariants.p}>
      {__(
        'Insights synchronization process is used to provide Insights recommendations output for hosts managed here.'
      )}
    </Text>
  </div>
);

export default InsightsHeader;
