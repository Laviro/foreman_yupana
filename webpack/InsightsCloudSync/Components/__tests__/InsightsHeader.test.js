import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import InsightsHeader from '../InsightsHeader';

const fixtures = {
  render: {},
};

describe('InsightsHeader', () =>
  testComponentSnapshotsWithFixtures(InsightsHeader, fixtures));
