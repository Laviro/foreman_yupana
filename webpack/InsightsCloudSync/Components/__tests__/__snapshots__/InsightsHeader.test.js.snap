// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsightsHeader render 1`] = `
<div
  className="insights-header"
>
  <Text
    component="p"
  >
    Insights synchronization process is used to provide Insights recommendations output for hosts managed here.
  </Text>
</div>
`;
